### PR TITLE
Add commands for editing and deleting jobs

### PR DESCRIPTION
**Pull request by chiralcentre**

### CODE REVIEW

File: src/main/java/seedu/hirehub/logic/commands/EditJobCommand.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    public static class EditJobDescriptor {
+        private String title;
+        private String description;
+        private Integer vacancy;
+
+        public EditJobDescriptor() {}
+
+        /**
+         * Copy constructor.
+         * A defensive copy of {@code tags} is used internally.
```
Comment (logical-1985516) [inline src/main/java/seedu/hirehub/logic/commands/EditJobCommand.java:128]: Should "tags" appear here?

**Review by logical-1985516 — APPROVED** [review]
Other than a nit, LGTM!

File: src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditJobCommand.MESSAGE_USAGE), pe);
+        }
+
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_TITLE, PREFIX_DESCRIPTION, PREFIX_VACANCY);
+
+        EditJobDescriptor editJobDescriptor = new EditJobDescriptor();
+
+        if (argMultimap.getValue(PREFIX_TITLE).isPresent()) {
+            editJobDescriptor.setTitle(ParserUtil.parseTitle(argMultimap.getValue(PREFIX_TITLE).get()));
```
[the remark this answers is not in the document]
Comment (logical-1985516) [reply src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java:42]: I think this is implemented in lines 66-67 of EditJobCommand.

File: src/main/java/seedu/hirehub/logic/commands/EditJobCommand.java (hunk 1 above)
Comment (chiralcentre) [reply src/main/java/seedu/hirehub/logic/commands/EditJobCommand.java:128]: fixed, thanks for highlighting

File: src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (chiralcentre) [reply src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java:42]: @logical-1985516 is correct

File: src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+     */
+    public EditJobCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TITLE,
+                PREFIX_DESCRIPTION, PREFIX_VACANCY);
+
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(argMultimap.getPreamble());
```
[the remark this answers is not in the document]
Comment (chiralcentre) [reply src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java:32]: yep I am editing based on index

File: src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (acekhoon) [reply src/main/java/seedu/hirehub/logic/parser/EditJobCommandParser.java:42]: okie

**Pull request merged**